x-pack/plugins/observability_solution/investigate/common/types.ts

```diff
@@ -5,47 +5,9 @@
  * 2.0.
  */
 
-import type { DeepPartial } from 'utility-types';
-
 export interface GlobalWidgetParameters {
   timeRange: {
     from: string;
     to: string;
   };
 }
-
-export interface Investigation {
```

Comment:
Is this completely removed or is it moved somewhere else? I couldn't find it being defined anywhere else.

Reply:
It's coming from the shared schema, using the InvestigationResponse